src/networking/payload/bits.rs

```diff
@@ -0,0 +1,28 @@
+
+#[derive(PartialEq, Debug)]
+pub struct Bits{
```

Comment:
What does this exactly do?

Reply:
makes the struct printable and comparable (all elements of the struct need to be equal)